distribution/lib/Standard/AWS/0.0.0-dev/src/AWS_Credential.enso

```diff
@@ -0,0 +1,30 @@
+from Standard.Base import all
+
+from Standard.Base.Metadata import Widget, make_single_choice
+
+polyglot java import org.enso.aws.ProfileReader
+
+type AWS_Credential
+    ## Access using IAM via an AWS profile.
+
+       Arguments:
+       - profile: AWS profile name (if empty uses default).
+    @profile (make_single_choice AWS_Credential.profile_names)
+    Profile profile:Text='default'
+
+    ## Access using IAM via an AWS access key ID and secret access key.
+
+       Arguments:
+       - access_key: AWS access key ID.
+       - secret_access_key: AWS secret access key.
+    Key access_key:Text secret_access_key:Text
+
+    ## Get a list of the available profiles
+    profile_names : Vector Text
+    profile_names = ProfileReader.INSTANCE.getProfiles
```

Comment:
```suggestion
    profile_names = 
    	Vector.from_polyglot_array <|
    		ProfileReader.INSTANCE.getProfiles
```

Don't we have to wrap this?

IIRC the result will have type `Array` otherwise, but we say we return `Vector`, so to be consistent with the signature, I think we need this wrapper.